src/components/modules/InfoBox/index.tsx

```diff
@@ -1,35 +1,39 @@
 import React, { PureComponent } from 'react';
-import { StyleSheet, View, Text, ImageBackground, Dimensions } from 'react-native';
+import { StyleSheet, View, Text, ImageBackground } from 'react-native';
 import { ModuleBox } from '@/components/modules';
 import { Image, Container, HtmlText } from '@/components/primitives';
 import { Responsive } from '@/components/utils';
 import { COLOR_MAP, BREAKPOINTS } from '@/constants';
 import { InfoBoxData, OFFSET_DIRECTION } from '@/types';
 
-interface Props {
-    data: InfoBoxData[];
+interface Props extends InfoBoxData {
+    name: string;
+    imageSrc: string;
+    description: string;
 }
 
 type State = {
     isExpanded: boolean;
-    width: number;
     height: number;
-    windowWidth: number;
-    imgHeight: number;
+    paginate: boolean;
```

Comment:
I think it'd be better if paginate was a property to infobox as opposed to a state. fp_data is going to be generated by WordPress, so it will be able to pass a property into the module definition. 

Maybe you could make a isPaginated member function that checks if isDevEnv, something like:
`isPaginated = () => isDevEnv() ? window.location.pathname.includes('/page/') : this.props.paginate;`